bayesian/utility/db_gateway.py

```diff
@@ -99,6 +103,30 @@ def get_batch_ca_data(cls, ecosystem: str, packages) -> dict:
         return response.json()
 
 
+def get_user_details(uuid: str) -> Optional[Dict]:
+    """Query RDS DB on user_details table.
+
+    :param uuid: UUID received in request.
+    :return: User Data
+    """
+    try:
+        start = time.time()
+        query = rdb.session.query(UserDetails) \
+            .filter(UserDetails.user_id == uuid)
+        user_details = query.one()
+        logger.info('%s took %f seconds to fetch data', uuid, time.time() - start)
+        return user_details.to_dict()
+
+    except (NoResultFound, MultipleResultsFound):
+        logger.info('No Result found.')
+        return None
+
+    except SQLAlchemyError as e:
+        rdb.session.rollback()
+        logger.exception('%s Error updating log, exception %s', uuid, str(e))
+        raise SQLAlchemyError('Error while checking uuid {}'.format(uuid)) from e
```

Comment:
I guess @dgpatelgit is also working on similar concept. Can you guys agree on a common approach to get user registration status?

Reply:
+1

Reply:
@arajkumar Okay will do.  
Why it's a **Requested change** instead of comment..?